FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+MessageSend.swift

```diff
@@ -41,11 +40,48 @@ extension ComposeViewController {
         handleSuccessfullySentMessage()
     }
 
+    internal func requestMissingPassPhraseWithModal(for signingKey: Keypair, isDraft: Bool = false) {
+        Task {
```

Comment:
`Task` is not needed here, as no async operations are performed

Reply:
You are right 👍